custom_components/airthings_wave/airthings.py

```diff
@@ -86,6 +87,21 @@ def decode_data(self, raw_data):
         return data
 
 
+class Wave2Decode(BaseDecode):
+    def decode_data(self, raw_data):
+        val = super().decode_data(raw_data)
+        val = val[self.name]
+        # if val[0] != 1:
```

Comment:
Just being a stickler here. Should this dead code be removed? Any reason to keep it in a comment?

Reply:
Yes it can be removed, i just forgot.